src/Uno.UWP/UI/Composition/SkiaCompositionSurface.skia.cs

```diff
@@ -62,7 +62,7 @@ internal void LoadFromBytes(byte[] image)
 
 		internal unsafe void SetPixels(int pixelWidth, int pixelHeight, byte[] data)
 		{
-			var info = new SKImageInfo(pixelWidth, pixelHeight, SKColorType.Rgba8888, SKAlphaType.Premul);
+			var info = new SKImageInfo(pixelWidth, pixelHeight, SKColorType.Bgra8888, SKAlphaType.Premul);
```

Comment:
What is the reason for this change?

Reply:
It was incorrect from the original implementation, as we're not automated testing this yet, so it got missed.